package.json

```diff
@@ -6,7 +6,6 @@
   "license": "MIT",
   "description": "Custom GitHub Action to deploy frontend apps to Contentful.",
   "main": "index.js",
-  "type": "module",
```

Comment:
Because of this issue 
https://github.com/contentful/ai-image-generator/actions/runs/3958526117/jobs/6780203268
and this issue is coming because of this line 
https://github.com/contentful/actions-app-deploy/blob/feat/branch-name-tracking/src/index.ts#L4

When ncc is bundling the code, it is converting to `require` 🤷

Reply:
So, one of the new packages is not compatible with ESM... great